### PR TITLE
Remove variables and references to variables of the AB test parameter - Remerge

### PR DESCRIPTION
**Pull request by jdc91**
This is a remerge of the previously reverted code at #45615

#### Changes done in the relevant Pull Request #44961
* Remove prop isEligibleVariantForDomainTest
* Further cleanup of dead code and minor style fix in domain suggestion
* Removed unnecessary parameter and import
* Fixed issue where the plans selection link was broken due to a missing dependency
* Hid the domain explainer based on whether the plans selection is available in the flow

#### Changes done in additional to initial PR at #44961
* Removed unnecessary check when providing dependencies from the domain step which breaks the contract with the plan step. This was the reason why the build pipeline was breaking which led to the revert at #45615. Test skipped, was reverted since the issue is now fixed.
* A parameter was invalidly specified in dependencies when should have been a "providesDependencies"
* Disable unnecessary Eslint rule which was breaking the team city pipeline

#### Additional Functional Tests Required due to changes in current code.
-  Check if hideFreePlan works properly check  #37554 for how to test
- Some key flows that are effected and were tested

<details>
  <summary>Click to expand!</summary>
  
  ```javascript
[
    {
        "flow": "business",
        "description": "Create an account and a blog and then add the business plan to the users cart.",
        "steps": [
            "user",
            "domains",
            "plans-business"
        ]
    },
    {
        "flow": "premium",
        "description": "Create an account and a blog and then add the premium plan to the users cart.",
        "steps": [
            "user",
            "domains",
            "plans-premium"
        ]
    },
    {
        "flow": "personal",
        "description": "Create an account and a blog and then add the personal plan to the users cart.",
        "steps": [
            "user",
            "domains",
            "plans-personal"
        ]
    },
    {
        "flow": "free",
        "description": "Create an account and a blog and default to the free plan.",
        "steps": [
            "user",
            "domains"
        ]
    },
    {
        "flow": "with-theme",
        "description": "Preselect a theme to activate/buy from an external source",
        "steps": [
            "domains-theme-preselected",
            "plans",
            "user"
        ]
    },
    {
        "flow": "onboarding",
        "description": "Abridged version of the onboarding flow. Read more in pau2Xa-Vs-p2.",
        "steps": [
            "user",
            "domains",
            "plans"
        ]
    },
    {
        "flow": "desktop",
        "description": "Signup flow for desktop app",
        "steps": [
            "about",
            "themes",
            "domains",
            "plans",
            "user"
        ]
    },
    {
        "flow": "ecommerce",
        "description": "Signup flow for creating an online store with an Atomic site",
        "steps": [
            "user",
            "domains",
            "plans-ecommerce-fulfilled"
        ]
    },
    {
        "flow": "domain",
        "description": "An experimental approach for WordPress.com/domains",
        "steps": [
            "domain-only",
            "site-or-domain",
            "site-picker",
            "themes",
            "plans-site-selected",
            "user"
        ]
    },
    {
        "flow": "add-domain",
        "description": "An approach to add a domain via the all domains view",
        "steps": [
            "select-domain",
            "site-or-domain",
            "site-picker",
            "themes",
            "plans-site-selected",
            "user"
        ]
    },
    {
        "flow": "launch-site",
        "description": "A flow to launch a private site.",
        "steps": [
            "domains-launch",
            "plans-launch",
            "launch"
        ]
    },
    {
        "flow": "import",
        "description": "A flow to kick off an import during signup",
        "steps": [
            "user",
            "from-url",
            "domains",
            "plans-import"
        ]
    },
    {
        "flow": "import-onboarding",
        "description": "Import flow that can be used from the onboarding flow",
        "steps": [
            "user",
            "site-type",
            "import-url",
            "import-preview",
            "domains",
            "plans-import"
        ]
    },
    {
        "flow": "new-launch",
        "description": "Launch flow for a site created from /new",
        "steps": [
            "domains-launch",
            "plans-launch",
            "launch"
        ]
    }
]
  ```
</details>

Test Plan by @niranjan-uma-shankar 

Consolidating all the points from the review comments, I'm listing below the flows to be tested and scenarios:

/start, onboarding flow:
(i) The new domain UI should be visible. This is currently the behaviour in production, and we want the same behaviour here.
(ii) The free domain explainer should be visible. Clicking "Review our plans to get started »" should take to the plans step with the "Start with free" option hidden.
(iii) Selecting the free *.wordpress.com subdomain should show the "Start with free" option in the plans step
(iv) Selecting a custom domain should hide the "Start with free" option in the plans step
(v) Try a combination of above options and verify they work - first click "Review our plans ..." -> free plan option hidden -> hit back button on the plans page -> select a free wordpress.com subdomain -> the free plan option should be visible -> hit back button on plans page -> select a custom domain and verify that the free plan option is hidden.

While logged in to an account, click "Add new site" and verify that all scenarios mentioned in the previous point work.

Plan specific flows, /start/personal, and also /premium, /business, /ecommerce:
(i) The new domain UI should be visible.
(ii) Free domain explainer should not be visible
(iii) Selecting either a free subdomain or custom domain should take to checkout(same as in prod)

/start/domain, domain-only flow:
(i) The new domain UI should not be visible.

/start/add-domain which is similar to the domain-only flow:
(i) The new domain UI should not be visible.

/start/launch-site which is the site launch flow:
(i) When the launch flow opens, the domain step already renders initial suggestions. The initial suggestions should not show the striked out text.
(ii) When searching for a domain in the launch flow, show the new domain UI
(iii) The free domain explainer is to be shown if the site is a on a free plan, else hide the free domain explainer if the site is on a paid plan.

Update

Since there are some bugs to iron out for the launch flow, we will deprioritize launch flow UI for now and take it as a follow up. So, point number 6 above can be ignore, and we can maintain status quo on launch flow UI.

### CODE REVIEW

File: client/signup/config/steps-pure.js
Comment (niranjan-uma-shankar) [inline client/signup/config/steps-pure.js:114]: This is incorrect - `shouldHideFreePlan` should be in `providesDependencies` and not `dependencies`. So, launch flow will not work.

File: client/state/signup/preview/selectors.js
Comment (niranjan-uma-shankar) [inline client/state/signup/preview/selectors.js:26]: Why is this selector under signup/preview/selectors.js file? Shouldn't be in a different folder since isPlanStepExistsAndSkipped Is not related to site previews?

**Review by niranjan-uma-shankar — CHANGES_REQUESTED** [review]

File: client/signup/config/steps-pure.js
Comment (jsnajdr) [inline client/signup/config/steps-pure.js:104]: Hi @jdc91 :wave: This change is different from what was in #44961, is that right?

In #44961, the `shouldHideFreePlan` was added to `dependencies`, not to `providesDependencies`. But that was making the flow dependencies invalid and the [_assertFlowHasValidDependencies](https://github.com/Automattic/wp-calypso/blob/2f156ae44e2c4d6151699e8e253e5d93d0657cc6/client/lib/signup/flow-controller.ts#L165) check would fail.

That's because the `domains-launch` step depends on the `shouldHideFreePlan` value, but it's not provided by anyone -- not in the query string (like `siteSlug`), nor by any other step in the flow. The `domains-launch` step actually _provides_ the value for other steps to consume.

The result of the failed validation is that the `launch-site` flow is redirected to the default flow (`/start`), breaking completely away from the expected UI.

The visible result is that instead of seeing `domains-launch` UI with a "Skip purchase" option, as the e2e test expects:

Screenshot 2020-09-15 at 10 30 37

I am redirected to `/start` and then to `/start/domains` (the first step of the default flow), seeing this:

Screenshot 2020-09-15 at 11 44 16

That's exactly what we've seen in the failed e2e tests.

The `_assertFlowHasValidDependencies` check verifies if the dependency is present in the dependency store, which can be loaded from local storage on boot. That explains the intermittency of the failures that @scinos observed: sometimes `shouldHideFreePlan` was already in local storage from previous run of the `domains-launch` step and the validation succeeded.

File: client/signup/steps/domains/index.jsx
Comment (jsnajdr) [inline client/signup/steps/domains/index.jsx:244]: This code looks suspicious and could be another source of mysterious issues.

The `domains` and `domains-launch` steps that use this component declare that they provide the `shouldHideFreePlan` dependency. Therefore they should always provide it, even if the value is `false`, when calling `submitStep`.

But here, the dependency is submitted only when `getIsPlanSelectionUnavailableInFlow()` is `true`. That could lead to another `_assertFlowHasValidDependencies` fail, because the `domains-launch` step didn't fulfill its contract.

Additionally, the `getIsPlanSelectionUnavailableInFlow()` seems to be reverted: it submits the `shouldHideFreePlan` dependency only if there is no `plans*` step in the flow. But the `plans*` steps are the only ones that actually consume the `shouldHideFreePlan` dependency. And they won't get it.

File: client/signup/config/steps-pure.js
Comment (jdc91) [reply client/signup/config/steps-pure.js:104]: Yes @jsnajdr this was the breaking change as mentioned above by @niranjan-uma-shankar. Thanks for the lengthy explanation why it was intermittent. 👍

File: client/signup/steps/domains/index.jsx
Comment (jdc91) [reply client/signup/steps/domains/index.jsx:244]: From what I gathered this means that we can remove this check completely from the  code and we actually should always pass this object either way. @niranjan-uma-shankar Do you agree?

Comment (jdc91) [reply client/signup/steps/domains/index.jsx:244]: And you care correct @jsnajdr the condition should be the other way around in any case.

File: client/signup/config/steps-pure.js
Comment (jsnajdr) [reply client/signup/config/steps-pure.js:104]: I still don't fully understand why the 'launch site' e2e test failed even after #44961 was reverted, but I suspect it has something to do with the conditional submission of the `{ shouldHideFreePlan }` dependency. That seems wrong and should be fixed before merging.

I looked at the e2e failures in this PR and they seem completely unrelated. Related to Media and Gutenberg. No problem merging the PR despite them.

File: client/signup/steps/domains/index.jsx
Comment (jsnajdr) [inline client/signup/steps/domains/index.jsx:244]: The `useThemeHeadstart` is another place that could simply always return the `{ useThemeHeadstart: boolean }` value no matter if `true` or `false`.

Otherwise, as this component is also used by the `domains-theme-preselected` step, that step's `providesDependencies: [ 'useThemeHeadstart' ]` contract is not always honored.

File: client/signup/steps/domains/index.jsx
Comment (jsnajdr) [inline client/signup/steps/domains/index.jsx:278]: This can be further simplified to
```js
Object.assign( { domainItem, shouldHideFreePlan }, useThemeHeadstartItem )
```
or even
```js
{ domainItem, shouldHideFreePlan, useThemeHeadstartItem }
```

**Review by jsnajdr — APPROVED** [review]
This should work :+1: Thanks for addressing the feedback.

I think that the `useThemeHeadstart` flag should be also improved, but that's not strictly necessary to merge this.

File: client/signup/steps/domains/index.jsx
Comment (jdc91) [reply client/signup/steps/domains/index.jsx:278]: I think its best to address in a seperate PR @jsnajdr 🙂 
Will get on this after merging the current PR if that's ok with you.

Comment (jsnajdr) [reply client/signup/steps/domains/index.jsx:278]: I agree that a separate PR is better :+1: This one is already big enough and as we've seen, a bit fragile and risky.

File: client/signup/config/steps-pure.js
Comment (niranjan-uma-shankar) [inline client/signup/config/steps-pure.js:327]: I don't think we should force `shouldHideFreePlan ` as a `providesDependencies` in every version of the domain step. Take the above line as example, in the domain-only flow it does not look relevant to send a `shouldHideFreePlan` dependency when we don't even have a plans step if the user just wants to buy a domain. This is not a bug, but it could just be confusing to developers in the future if someone wants to add a domain step and wondering why that step should forced to send a prop related to plans.

Going back to [this line](shouldHideFreePlan) which was removed in this PR, every new flow in the future that includes a domain step will be forced to add `shouldHideFreePlan` as a dependency. A problem is that this tightly couples the order of steps and assumes that the plans step will either always be present in the flow, or will always appear _after_ the domains step. What if we have a new flow that shows the plans step _before_  the domains step?(example, check the A/B test flow [onboarding-plan-first](https://github.com/Automattic/wp-calypso/blob/master/client/signup/config/flows-pure.js#L125). Then `shouldHideFreePlan` as a provided dependency in the domain step doesn't look right if the plan step was already completed by the time the domain step loaded in the hypothetical flow. Again, not a bug but it just logically doesn't fit. 

I think it's probably better to keep the existing pattern of adding `shouldHideFreePlan` as an optional dependency([check here how the domain step does it](https://github.com/Automattic/wp-calypso/blob/master/client/signup/config/steps-pure.js#L318)), and retain the lines that were [removed here](https://github.com/Automattic/wp-calypso/pull/45618/files#diff-df013e2a4c6a6c2b5973d50e326984f8L258). We can then completely remove `shouldHideFreePlan` as a dependency for "domain-only" and "select-domain" steps.

**Review by niranjan-uma-shankar — CHANGES_REQUESTED** [review]
The free domain explainer needs to be hidden for the domain-only flow since the content in that explainer is not relevant in this flow. There is no paid plan selection in the next step. Also. clicking "Review our plans to get started »" is not working.

signup

I suggest that we not show the free domain explainer in the domain-only or add-domain flows. Probably you can do this with a prop value in steps-pure.js for the domain step used in those flows, [example here](https://github.com/Automattic/wp-calypso/blob/master/client/signup/config/steps-pure.js#L330).

File: client/signup/steps/domains/index.jsx
Comment (jdc91) [reply client/signup/steps/domains/index.jsx]: Fixed with fb4b0a313581a96c0093e6bd61309326c25343a3

File: client/signup/config/steps-pure.js
Comment (jdc91) [reply client/signup/config/steps-pure.js:327]: @niranjan-uma-shankar Can you please fix the link at "this line" in the following sentence. It seems to be incorrect.
> Going back to this line which was removed in this PR, every new flow in the future that includes a domain step will be forced to add shouldHideFreePlan as a dependency.

Comment (niranjan-uma-shankar) [reply client/signup/config/steps-pure.js:327]: Done

Comment (jdc91) [reply client/signup/config/steps-pure.js:327]: I still think we do not need to bring back the code removed [here](https://github.com/Automattic/wp-calypso/pull/45618/files#diff-df013e2a4c6a6c2b5973d50e326984f8L258), since there is anyway a default parameter on the function and we will fallback to it being false whenever there is no value provided. 

However I will move the dependency to optional dependencies so that we don't make this step force to expect that parameter.

Comment (jdc91) [reply client/signup/config/steps-pure.js]: Fixed at e2bc89cefa6b22738452a6a0eabd8d60e2e8b0af

File: client/signup/steps/domains/index.jsx
Comment (jdc91) [reply client/signup/steps/domains/index.jsx]: Code refactored with this fixed.

File: client/components/domains/register-domain-step/index.jsx
Comment (niranjan-uma-shankar) [inline client/components/domains/register-domain-step/index.jsx:1166]: For a site on the free plan, when I entered the launch flow, the initial suggestions view looks like this:

Screenshot 2020-09-23 at 12 38 42 PM

Notice the "Registration free" text for the domain price. The problem with showing this without the free domain explainer is that the domain price is misleading. The registration is free only if a paid plan is selected and this is explained in the free domain explainer box, but in the screenshot above it seems to indicate that registration is always free. We should either show the free domain explainer, or change the domain price text and I prefer the latter since showing the explainer in this view may not aesthetically fit in.

In the above line which is in `renderInitialSuggestions()`, can we add another prop to indicate this is an initial suggestion? Maybe something like `isInitialSuggestion={ true }`, and then [around here](https://github.com/Automattic/wp-calypso/pull/45618/files#diff-a9690a078a161112454a9d1639b75a58R48) you would show the registration free text only if `isSignupStep && ! isInitialSuggestion`. Maybe there is a better way to do it, just saying something top of my mind. 




can we add a prop isPlanSelectionUnavailableInFlow={ isPlanSelectionUnavailableInFlow } too? Then , render the "registration free" text only if plan selection

Comment (niranjan-uma-shankar) [inline client/components/domains/register-domain-step/index.jsx:1166]: For a site on a paid plan, when I enter the launch flow, the initial suggestions and search results look like this:

Screenshot 2020-09-23 at 12 47 46 PM

Screenshot 2020-09-23 at 1 00 56 PM

The domain price should not be listed there, instead we should see the "Registration free" text here since the user is already on a paid plan. I noticed that this bug exists on production too, so you don't need to fix it in this PR. If you prefer to tackle it as a separate PR, could you please track the issue in a new GH issue? Thanks.

**Review by niranjan-uma-shankar — COMMENTED** [review]
Thanks for working through the suggestions! I tested this and left a few comments below.

**Review by niranjan-uma-shankar — COMMENTED** [review]
I just noticed that the launch flow does not display the free domain explainer if I cancel a paid plan. 

Steps to follow:
1. Create a new site on a paid plan. Confirm your account by following the link sent to your email inbox.
2. Go to /me/purchases and cancel your paid plan, so now you will be on a free plan.
3. Open the launch flow and search for a domain.
4. Notice the missing free domain explainer

Screenshot 2020-09-23 at 1 37 46 PM

**Comment by jdc91** [comment]
@niranjan-uma-shankar RE:
> I just noticed that the launch flow does not display the free domain explainer if I cancel a paid plan.

This is the behaviour in prod as well, should we change this? Even if so should it be in a separate PR?

File: client/components/domains/register-domain-step/index.jsx
Comment (jdc91) [inline client/components/domains/register-domain-step/index.jsx:614]: The key change is here

File: client/signup/steps/domains/index.jsx
Comment (jdc91) [reply client/signup/steps/domains/index.jsx:297]: This is tracked under : https://github.com/Automattic/martech/issues/638

File: client/components/domains/register-domain-step/index.jsx
Comment (jdc91) [reply client/components/domains/register-domain-step/index.jsx:1180]: Fixed at commit :  https://github.com/Automattic/wp-calypso/pull/45618/commits/ed504ab41af752dbbfedc8bc6e2e8b512a04759e#diff-99f1f1d8ba147cd06cfd4a0d8d195d34R592-R614

Comment (jdc91) [reply client/components/domains/register-domain-step/index.jsx:1180]: Tracked @ 639-gh-martech

File: client/signup/steps/domains/index.jsx
Comment (niranjan-uma-shankar) [reply client/signup/steps/domains/index.jsx:297]: > This will be fixed with : #45997

Thanks for taking on that refactor. I think that users going back and forth between steps could be common, so we would need this issue fixed before merging the current PR.

How do you see us proceeding on this issue? Should we wait till #45997 is reviewed and deployed? Another option  is to fix the issue without waiting for the framework level changes so that we are not blocked. The framework level change can then be reviewed separately.

If we want to fix this locally so that we are not blocked by a review for another PR, I think we can simply set the dependency like so:


```
shouldHideFreePlan  || this.getIsPlanSelectionAvailableInFlow() ? { shouldHideFreePlan } : {},
```

Hope you don't mind but I just found it easier to explain my thoughts with code.

Please let me know what you think.

Comment (jdc91) [reply client/signup/steps/domains/index.jsx:297]: I played around with this and @niranjan-uma-shankar and the problem here is that  ``` this.getIsPlanSelectionAvailableInFlow()``` will be true for domain only flows like the following flows. 
image

As shown in the [implementation of getIsPlanSelectionAvailableInFlow](https://github.com/Automattic/wp-calypso/pull/45618/files#diff-df013e2a4c6a6c2b5973d50e326984f8R133-R148).  we only check if a plans aliased step appears anywhere in the flow.

So this will result in a parameter for flows where we have removed the dependency steps flows pure. 

We can try to do this change by mandating that any domains step should always provide this parameter. However I think you argued against this contradiction [already](https://github.com/Automattic/wp-calypso/pull/45618#discussion_r491830138)

However we can do something if there was an actual way to figure out, **which steps are "dead or dummy"** even if they appears in the flow schema at flows pure. This way we can truly figure out which steps are actually visible to the user. This together with checking whether a plans step occurs after the domains step we might be able to change getIsPlanSelectionAvailableInFlow](https://github.com/Automattic/wp-calypso/pull/45618/files#diff-df013e2a4c6a6c2b5973d50e326984f8R133-R148
) and provide this logic

Comment (niranjan-uma-shankar) [reply client/signup/steps/domains/index.jsx:297]: > and the problem here is that this.getIsPlanSelectionAvailableInFlow() will also be true for domain only flows like the following flows

Good point. Would it work if we use `forceHideFreeDomainExplainer` prop that is defined [here](https://github.com/Automattic/wp-calypso/pull/45618/files#diff-ae0bf020974ae29c034739f287afc5c5R341) to determine if the `shouldHideFreePlan` dependency should be added for the current step?

Comment (niranjan-uma-shankar) [reply client/signup/steps/domains/index.jsx:297]: So, it would be:

```
! forceHideFreeDomainExplainer  && ( shouldHideFreePlan  || this.getIsPlanSelectionAvailableInFlow() ) ? { shouldHideFreePlan } : {},
```

The above conditional could maybe be better organized for readability, but just quickly typed something to convey the concept.

Comment (jdc91) [reply client/signup/steps/domains/index.jsx]: I did come up with this change https://github.com/Automattic/wp-calypso/pull/45618/commits/ed869f91f8dda3738b26967e57800c3bb39e8cb6. Let me know what you think @niranjan-uma-shankar

Comment (niranjan-uma-shankar) [reply client/signup/steps/domains/index.jsx]: This seems to be on the right track. I'll test it out shortly.

**Comment by niranjan-uma-shankar** [comment]
Consolidating all the points from the review comments, I'm listing below the flows to be tested and scenarios:

1. /start, `onboarding` flow:
   (i) The new domain UI should be visible. This is currently the behaviour in production, and we want the same behaviour here.
   (ii) The free domain explainer should be visible. Clicking "Review our plans to get started »" should take to the plans step with the "Start with free" option hidden.
   (iii) Selecting the free *.wordpress.com subdomain should show the "Start with free" option in the plans step
   (iv) Selecting a custom domain should hide the "Start with free" option in the plans step
   (v) Try a combination of above options and verify they work - first click "Review our plans ..." -> free plan option hidden -> hit back button on the plans page -> select a free wordpress.com subdomain -> the free plan option should be visible -> hit back button on plans page -> select a custom domain and verify that the free plan option is hidden.

2. While logged in to an account, click "Add new site" and verify that all scenarios mentioned in the previous point work.

3. Plan specific flows, /start/personal, and also /premium, /business, /ecommerce:
    (i) The new domain UI should be visible.
    (ii) Free domain explainer should not be visible 
    (iii) Selecting either a free subdomain or custom domain should take to checkout(same as in prod)

4. /start/domain, domain-only flow: 
    (i) The new domain UI should not be visible.

5. /start/add-domain which is similar to the domain-only flow:
    (i) The new domain UI should not be visible.

6. /start/launch-site which is the site launch flow:
    (i) When the launch flow opens, the domain step already renders initial suggestions. The initial suggestions should not show the striked out text.
    (ii) When searching for a domain in the launch flow, show the new domain UI
    (iii) The free domain explainer is to be shown if the site is a on a free plan, else hide the free domain explainer if the site is on a paid plan.

**Review by niranjan-uma-shankar — APPROVED** [review]
I pushed in a [small commit](https://github.com/Automattic/wp-calypso/pull/45618/commits/a3a1bfe79d3b2b611fbceb527064b0c12c30b4b5) that removes `isSupressedDependencyShouldHideFreePlan`, since with just `forceHideFreeDomainExplainerAndStrikeoutUi` we can still determine if we want to suppress the `shouldHideFreePlan` dependency.

I tested all signup flows that have a domain - onboarding, plan-specific flows, with-theme - and confirm that I can see the new domain UI. Confirmed that the plan-sepcific flows will not have the free domain explainer.

The launch site flow works well, and displays the new domain UI. 

The Manage domains section in Calypso displays the old UI and this is acceptable since we are only changing the UI for the signup flows. The domain-only and addd-domain flows display the old UI as expected, since these flows don't have a plan step right after the domain step.

LGTM, but we will hold from deploy until the currently running signup tests finish running.

File: client/signup/steps/domains/index.jsx
Comment (scinos) [inline client/signup/steps/domains/index.jsx:4]: Not sure why this was changed, it will be nice to maintain previous order so it is easier to review.

Comment (jdc91) [reply client/signup/steps/domains/index.jsx:4]: Hmmm there was a merge conflict and I just vscode formatted imports :(
I'll see if I can fix this.
CC - @niranjan-uma-shankar

Comment (jdc91) [reply client/signup/steps/domains/index.jsx]: Fixed

**Review by niranjan-uma-shankar — APPROVED** [review]
Re-tested and LGTM 🚢 

Noting that the "Select" button gets renamed to "Upgrade" in the domain-only flow, and in the launch flow initial suggestions list. It's a minor fix, and we can take this in a follow-up PR. 
Here's how it looks in this branch - [domain-only screenshot](https://d.pr/i/SIq7wF), [launch-site screenshot](https://d.pr/i/T2AC1e)
Here's how it looks in production - [domain-only screenshot](https://d.pr/i/y0Ntj8), [launch-site screenshot](https://d.pr/i/y0Ntj8)

**Pull request merged**
